src/proteomes/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,365 @@
+            <Link
+              to={generatePath(LocationToPath[Location.HelpEntry], {
+                accession: 'taxonomic_identifier',
+              })}
+            >
+              taxonomy identifier
+            </Link>
+            .
+          </p>
+          <p>
+            UniProt proteomes may include both manually reviewed
+            (UniProtKB/Swiss-Prot) and unreviewed (UniProtKB/TrEMBL) entries.
+            The proportion of reviewed entries varies between proteomes, and is
+            greater for the proteomes of intensively curated model organisms.
+            Some proteomes, such as those of{' '}
+            <Link
+              to={{
+                pathname: LocationToPath[Location.UniProtKBResults],
+                search: stringifyQuery({ query: 'proteome:UP000002311' }),
+              }}
+            >
+              Saccharomyces cerevisiae 288C
+            </Link>{' '}
+            and{' '}
+            <Link
+              to={{
+                pathname: LocationToPath[Location.UniProtKBResults],
+                search: stringifyQuery({ query: 'proteome:UP000000625' }),
+              }}
+            >
+              Escherichia coli strain K12
+            </Link>{' '}
+            consist entirely of reviewed entries. Curation is a continuing
+            process, and proteomes are updated in a regular manner as new
+            information becomes available: pseudogenes and other dubious
+            uncharacterized ORFs may be removed, other newly identified and
+            characterized sequences may be added.
+          </p>
+          <p>
+            <Link
+              to={{
+                pathname: LocationToPath[Location.ProteomesResults],
+                search: stringifyQuery({ query: '*' }),
+              }}
+            >
+              Start searching in Proteomes »
+            </Link>
+          </p>
+        </div>
+
+        {/* Statistics */}
+        <section className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-9">
+          <h2>Statistics</h2>
+          <div className={styles.statistics}>
+            <div className={styles.chart}>
+              <h3 className="tiny">Taxonomic origin</h3>
+              <StatisticsChart
+                refProt={refProtHovered}
+                nonRefProt={nonRefProtHovered}
+              />
+            </div>
+            <section className={styles['entries-count']}>
+              <h2>Proteome Status</h2>
+              <h3 className="tiny">Number of Entries</h3>
+              <br />
+              <p
+                className={styles['stats-count']}
+                onPointerEnter={() => setRefProtHovered(true)}
+                onFocus={() => setRefProtHovered(true)}
+                onPointerLeave={() => setRefProtHovered(false)}
+                onBlur={() => setRefProtHovered(false)}
+              >
+                <ReferenceProteomeIcon
+                  width="2em"
```

Comment:
This appears to be a different size. Maybe because the underlying SVG has a different size so might need to adjust the viewbox, or maybe try a different unit for width (ch, rem?), or adjust one of the values in order to make them visually the same